README.md

```diff
@@ -3,3 +3,102 @@
 The official SurrealDB library for JavaScript.
 
 [![](https://img.shields.io/badge/status-beta-ff00bb.svg?style=flat-square)](https://github.com/surrealdb/surrealdb.js) [![](https://img.shields.io/badge/docs-view-44cc11.svg?style=flat-square)](https://surrealdb.com/docs/integration/libraries/javascript) [![](https://img.shields.io/badge/license-Apache_License_2.0-00bfff.svg?style=flat-square)](https://github.com/surrealdb/surrealdb.js)
+
+## Install
+
+### Deno
+Import it with
+
+```ts
+import Surreal from 'https://deno.land/x/surrealdb/mod.ts';
```

Comment:
We should add a note regarding using pinned modules for production, or add a separate example using pinned latest version (might need to be updated for each version bump though).